RELEASE_NOTES.md

```diff
@@ -1,5 +1,33 @@
 # Release Notes
 
+## stable-1.15-2.2.0, stable-1.16-2.2.0, stable-1.17-2.2.0
+
+* Prometheus
+  * Fix an issue that may cause Grafana's home dashboard to be empty. ([#351](https://github.com/mesosphere/kubernetes-base-addons/pull/351), [@branden](https://github.com/branden))
+  * bump chart to v9.3.1 ([#444](https://github.com/mesosphere/kubernetes-base-addons/pull/444), [@samvantran](https://github.com/samvantran))
+    * chore: bump prometheus to v2.18.2
+    * refactor!: (breaking change) version 9 of the helm chart removes the existing `additionalScrapeConfigsExternal` in favor of `additionalScrapeConfigsSecret`. This change lets users specify the secret name and secret key to use for the additional scrape configuration of prometheus.
+    * feat: add ingress configuration for Thanos sidecar, enabling external access from a centralized thanos querier running in another cluster
+    * feat: add scrape timeout config to service monitor to avoid timeouts on slow kubelets
+    * feat: add docker checksum option to improve security for deployed containers
+    * feat: add option to disable availability rules
+    * feat: enable scraping /metrics/resource for kubelet service
+    * feat: [prometheus] enable namespace overrides
+    * feat: [prometheus] allow additional volumes and volumeMounts
+    * chore: bump alertmanager to v0.21.0
+    * chore: bump grafana to v5.3.0
+    * feat: [alertmanager] add volume and volume mounts to spec
+    * feat: [alertmanager] add support for serviceAccount.annotations
+    * feat: [grafana] enable adding annotations to all default dashboard configmaps
+    * fix: add missing grafana annotations to k8s-coredns dashboard
+    * fix: reduced CPU utilization and time lag for code_verb:apiserver_request_total:increase30d scrape
+    * fix: invalid image pull policy for the admission webhook patch
+    * fix: alert "KubeNodeUnreachable" no longer fires on an autoscaling scale-down event
```

Comment:
Can we hold off this prometheus upgrade until we put it through soak?

Reply:
Ah, right... forgot that got merged that way. Fixing it.